app/views/tournament/side.scala

```diff
@@ -60,7 +60,9 @@ object side:
             markdownLinksOrRichText(d)
           ),
         tour.looksLikePrize option bits.userPrizeDisclaimer(tour.createdBy),
-        views.html.gathering.verdicts(verdicts, tour.perfType),
+        tour.isFinished option views.html.gathering.verdicts(verdicts, tour.perfType, false) orElse Some(
+          views.html.gathering.verdicts(verdicts, tour.perfType)
+        ),
```

Comment:
Looks like Swiss tournaments use verdicts in their view too. Would you be willing to make the change there too?

Reply:
I did not notice that, thank you for pointing that out, fixed that in the last commit!

On a side node, I also merged master back into this branch before making the changes, is that a problem? (kind of new to making PR's for open source projects)

Reply:
> On a side node, I also merged master back into this branch before making the changes, is that a problem? (kind of new to making PR's for open source projects)

It should be OK but you could squash your commits and force push to the branch if you want.

Reply:
I will look into squashing them this weekend, or would making a new pr from a new brench be easier?